crates/pop-parachains/templates/pallet/advanced_mode/src/benchmarking.rs.templ

```diff
@@ -0,0 +1,16 @@
+use super::*;
+
+#[allow(unused)]
+use crate::{frame_system::RawOrigin, Pallet, mock::{new_test_ext, Test}};
+use frame::deps::frame_benchmarking::v2::*;
```

Comment:
not sure if we have one, but we should make a `frame::benchmarking::prelude`, if a good one doesn't exist. 

Should have been part of https://github.com/paritytech/polkadot-sdk/pull/4406/files

Reply:
It seems there's not a `benchmarking` module in the frame crate yet: https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_frame/index.html?search=benchmarking

Reply:
I have to look at this a little bit more, `frame::benchmarking::prelude` isn't compiling, but maybe there's another way without using deps.